earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,116 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
```

Comment:
Minor: I think this file name would be better capitalized if it was `BUILD_RHEL_CentOS.md`.